app/settings.py

```diff
@@ -126,7 +126,7 @@ def utcoffset_or_fail(date_value, key):
 EQ_SESSION_ID = "eq-session-id"
 
 EQ_LIST_ITEM_ID_LENGTH = 6
-MAX_NUMBER = 9999999999
+MAX_NUMBER = 9999999999999
```

Comment:
Makes it easier to read with the `_` separator (it is only syntactic sugar). 
```suggestion
MAX_NUMBER = 9_999_999_999_999
```